agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -431,20 +441,27 @@ private static SdkLogEmitterProviderBuilder configureLogging(
       SdkLogEmitterProviderBuilder builder,
       TelemetryClient telemetryClient,
       @Nullable QuickPulse quickPulse,
+      ConfigProperties config,
       Configuration configuration) {
 
-    LogExporter logExporter = createLogExporter(telemetryClient, quickPulse, configuration);
-
-    // using BatchLogProcessor in order to get off of the application thread as soon as possible
-    batchLogProcessor =
-        BatchLogProcessor.builder(logExporter).setScheduleDelay(getBatchProcessorDelay()).build();
-
     // inherited attributes log processor also handles operation name, ikey and role name attributes
     // and these all need access to Span.current(), so must be run before passing off to the
     // BatchLogProcessor
-    return builder.addLogProcessor(
-        new InheritedAttributesLogProcessor(
-            configuration.preview.inheritedAttributes, batchLogProcessor));
+    builder.addLogProcessor(
+        new InheritedAttributesLogProcessor(configuration.preview.inheritedAttributes));
+
+    String logsExporter = config.getString("otel.logs.exporter");
+    if ("none".equals(logsExporter)) { // "none" is the default set in AiConfigCustomizer
```

Comment:
[here](https://github.com/microsoft/ApplicationInsights-Java/pull/2453/files#:~:text=is%20configured%20later%20in%20this%20case-,//%20this%20overrides%20the%20default%20%22otlp%22%20so%20the%20exporter%20can%20be%20configured%20later,-properties.put\(%22otel.logs) said 'otel' though.

Reply:
I didn't follw this comment, and the line numbers have changed since then, can you check this again?